packages/manager/cypress/e2e/core/kubernetes/lke-summary-page.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { mockGetCluster, mockGetKubeconfig } from 'support/intercepts/lke';
+import { kubernetesClusterFactory } from 'src/factories';
+import { readDownload } from 'support/util/downloads';
+
+const mockKubeconfigContents = '---'; // Valid YAML.
+const mockKubeconfigResponse = {
+  kubeconfig: btoa(mockKubeconfigContents),
+};
+
+const mockCluster = kubernetesClusterFactory.build();
+const url = `/kubernetes/clusters/${mockCluster.id}`;
+
+describe('LKE summary page', () => {
+  beforeEach(() => {
+    mockGetCluster(mockCluster).as('getCluster');
+    mockGetKubeconfig(mockCluster.id, mockKubeconfigResponse).as(
+      'getKubeconfig'
+    );
+  });
+
+  it('can download kubeconfig', () => {
+    const mockKubeconfigFilename = `${mockCluster.label}-kubeconfig.yaml`;
+    cy.visitWithLogin(url);
+    cy.wait(['@getCluster']);
+    cy.get(`p:contains(${mockKubeconfigFilename})`)
```

Comment:
```suggestion
    cy.findByText(mockKubeconfigFilename)
```
Small nit -- we can also use `cy.findByText` when clicking the "View" button in the other test